values/mainnet/network/api-read-slave-0.yaml

```diff
@@ -9,7 +9,7 @@ init:
   clearRestart:
     enabled: false
 
-Lotus:
+lotus:
```

Comment:
Are you sure we need to have this change?
I think we need to revert or make additional changes